sdk/core/test/integration/NetworkMember.test.ts

```diff
@@ -1094,4 +1094,49 @@ describe('CommonNetworkMember', () => {
     const commonNetworkMember = new AffinidiWallet(password, encryptedSeedElem, options)
     await commonNetworkMember.deleteAllCredentials()
   })
+
+  it('#createWallet should return wallet with resolvable did (elem)', async () => {
+    const commonNetworkMember = await AffinidiWallet.createWallet(
+      {
+        ...options,
+        didMethod: 'elem',
+      },
+      password,
+    )
+
+    const resolvedDidDocument = await commonNetworkMember.resolveDid(commonNetworkMember.did)
+
+    expect(resolvedDidDocument.id).equal(commonNetworkMember.didDocumentKeyId.split('#')[0])
```

Comment:
Can't we just compare it to `commonNetworkMember.did`?

Otherwise these tests will rely on implementation detail that `didDocumentKeyId` is combined of the did you're looking for, plus `#`, plus key name. And this detail changed just recently.

Reply:
> as for old elem with initial state the value of commonNetworkMember.did is not the same as resolvedDidDocumentkeyId

So `resolveDid` returns document with a id different from the did requested?

Reply:
it depends on what do you mean when saying "different"

they are different as strings  for old elem
id: _did:elem:EiCLgnTdCKKOoT_nxV2gDbfA0aw7AX4f5vGkZFsOuBX9DQ_
did: _did:elem:EiCLgnTdCKKOoT_nxV2gDbfA0aw7AX4f5vGkZFsOuBX9DQ_;elem:initial-state=eyJwcm90ZWN0ZWQiOiJleUp2Y0dWeVlYUnBiMjRpT2lKamNtVmhkR1VpTENKcmFXUWlPaUlqY0hKcGJXRnllU0lzSW1Gc1p5STZJa1ZUTWpVMlN5SjkiLCJwYXlsb2FkIjoiZXlKQVkyOXVkR1Y0ZENJNkltaDBkSEJ6T2k4dmR6TnBaQzV2Y21jdmMyVmpkWEpwZEhrdmRqSWlMQ0p3ZFdKc2FXTkxaWGtpT2x0N0ltbGtJam9pSTNCeWFXMWhjbmtpTENKMWMyRm5aU0k2SW5OcFoyNXBibWNpTENKMGVYQmxJam9pVTJWamNESTFObXN4Vm1WeWFXWnBZMkYwYVc5dVMyVjVNakF4T0NJc0luQjFZbXhwWTB0bGVVaGxlQ0k2SWpBek5EYzNOV1prTlROa01qUXpPR016WWpWaU9UYzBZamRqTkRjME9UWTROV0UyTkRrMk1UVTRZV1ExWVdVMk56Qm1aakZqWXpnNFpHRXdNVFUyTkdVeVpTSjlMSHNpYVdRaU9pSWpjbVZqYjNabGNua2lMQ0oxYzJGblpTSTZJbkpsWTI5MlpYSjVJaXdpZEhsd1pTSTZJbE5sWTNBeU5UWnJNVlpsY21sbWFXTmhkR2x2Ymt0bGVUSXdNVGdpTENKd2RXSnNhV05MWlhsSVpYZ2lPaUl3TWpZek5UVmlPV1l4TVdFM1lXUmlaR05qWlRSaE5EQXhZalUzTlRSbFptVmpNREkzTnpkbU5XSXdNalV6TUdFeE9ETXlNakUxWldNM01qRmhZelpoTkRBaWZWMHNJbUYxZEdobGJuUnBZMkYwYVc5dUlqcGJJaU53Y21sdFlYSjVJbDBzSW1GemMyVnlkR2x2YmsxbGRHaHZaQ0k2V3lJamNISnBiV0Z5ZVNKZGZRIiwic2lnbmF0dXJlIjoia1NKbGE5elNtYlNsUDg1WDU5bHRWaG5IUzFTVm90c0plR0tST0lTM2NHWVUzaHlzZC01ZGhFYVJidUNaN05TZ1ZaeEJFLWNFTjVDb2U5LXhSa0JUYmcifQ